Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/EventTag/ByEventTag.tsx

```diff
@@ -81,6 +82,12 @@ const EventTags: Application.Types.iByComponent = (props) => {
         setErrors(e);
     }, [record]);
 
+    const deleteTag = () => {
+        dispatch(EventTagSlice.DBAction({ verb: 'DELETE', record }));
+        setShowWarning(false);
+        setMode('View');
+    };
```

Comment:
```suggestion
    const deleteTag = React.useCallback(() => {
        dispatch(EventTagSlice.DBAction({ verb: 'DELETE', record }));
        setShowWarning(false);
        setMode('View');
    }, []);
```

This can be wrapped in React.useCallback to avoid regeneration on each render. Dependency array may be empty.